ftw/testbrowser/core.py

```diff
@@ -206,7 +208,7 @@ def open_html(self, html):
         :type html: string or file-like object
         :returns: The browser object.
         """
-        self.response = self._load_html(html)
+        self.response = self._load_html(html, subparser=self.parser)
```

Comment:
No i don't want it to be used always.  browser. parse_as_xml also uses _load_html with lxml.etree.parse as a parse function instead of lxml.html.parse but the parser we hand down still does its stuff if we hand a html parser to  lxml.etree.parse it still will work as a html parser and for instance make the root node a <html> tag. therefore we only want to pass it in certain cases which is why i pass it down so we only have it where we want.